### ISSUE TITLE
Quick questions on resources used for the full train

### ISSUE THREAD

**Issue by glmcdona** [aigc-apps/CogVideoX-Fun#103]
Couple quick questions if possible! Looking to do a full finetune as an experiment with a i2v dataset I've been working on.

* `train.sh`:
  * I note a 2e-5 learning rate. Was this the learning used for the full train of the model the entire time? Or was a learning rate schedule ran or something? Was 2e-5 the ending learning rate, or starting learning rate where it was lowered throughout the training? Trying to understand if this 2e-5 is a reasonable starting learning rate for a finetune.
  * train_batch_size of 1, does this mean a batch size of 1 was used? Should this be raised?
* Training resources
  * How many iterations/steps were ran?
  * Curious approximately how much the full train cost and how long with what resources?
